ui/admin/app/components/form/role/index.js

```diff
@@ -12,5 +12,6 @@ export default class FormRoleGrantsComponent extends Component {
   @action
   updateGrantScopeID(id) {
     this.args.model.grant_scope_id = id;
```

Comment:
I think it'd be clearer to add some TODO's that these are meant to be removed

Reply:
I can add this.